var/spack/repos/builtin/packages/rose/package.py

```diff
@@ -209,12 +215,16 @@ def configure_args(self):
 
         if "+cxx11" in spec:
             args.append("CXXFLAGS=-std=c++11")
+        if "+cxx14" in spec:
+            args.append("CXXFLAGS=-std=c++14")
```

Comment:
We need to use compiler specific flags for the recipe to be general, for instance `self.compiler.cxx14_flag`